src/views/Claims.vue

```diff
@@ -48,27 +63,39 @@ export default class Claims extends Vue {
     contract!: Contract;
     reward!: number;
     tokenAndAmount: Token[] = [];
-    error = false;
+    error = '';
+    walletExist = false;
+    loading = false;
     /**
      * Connects user to metamask, after that the reward variable is updated
      */
     async connect() {
+        const getWalletUrl = 'http://localhost:3001/v1/claims/';
```

Comment:
Move base URL to ENV, it is easier to change it in 1 location instead of having to search for multiple mentions.
The endpoint (/claims) will most likely remain the same.

Look at the for example the .env.local where VUE_APP_API_URL is present.